README.md

```diff
@@ -1,5 +1,7 @@
 <a name="readme-top"></a>
 ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+![Image overview](./public/img-view.jpg)
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
```

Comment:
- [ ] Dear @aubinleyoung, Good job you have done already. I notice you have not updated your acknowledgment section. Please remember to give an "Acknowledgements" and credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).